migrations/25.0/README.md

```diff
@@ -0,0 +1,117 @@
+# From 24.0 to 25.0
+
+## Header
+
+[Georchestra WebComponent Header](https://github.com/georchestra/header) can now be [configured](https://github.com/georchestra/header/blob/main/CONFIG.md) using a file at runtime.
+
+The default configuration is provided in [`default-config.json`](https://github.com/georchestra/header/blob/main/src/default-config.json).
+Here is [an example](https://github.com/georchestra/header/blob/main/public/sample-config.json) of file which can be used to configure the header.
+
+Each object configuration is set [here](https://github.com/georchestra/header/blob/main/src/config-interfaces.ts#L32-L53).
+
+The file must be provided by a web server (e.g. nginx) and must be configured in several files of the datadir:
+- `default.properties` -> [headerConfigFile](https://github.com/georchestra/datadir/blob/25.0/default.properties#L49)
+- `mapstore/config/localConfig.json` -> [header.configFile](https://github.com/georchestra/datadir/blob/docker-25.0/mapstore/configs/localConfig.json#L29)
+
+### Header in Metadata Editor
+
+The Metadata-editor, which comes from the Geonetwork-UI suite, needs a small adaptation to work with the geOrchestra header.
+
+Before the injection of the `<geor-header>` tag, we need to inject a CSS script (replace `<your-header-height>` with the height of your header):
+
+```css
+body {display: flex; flex-direction: column} 
+body md-editor-root, .h-screen {height: calc(100vh - <your-header-height>px);}
+main {max-height: calc(100vh - <your-header-height>px - 65px) !important}
+```
+
+## Elasticsearch and Kibana
+
+Since Geonetwork [4.4.3](https://docs.geonetwork-opensource.org/4.4/overview/change-log/version-4.4.3/#index-changes), Elasticsearch 8.x is supported. 
+
+### ES v7 to v8 upgrade
+
+Elasticsearch can be upgraded from v7 to v8.
+
+If so, you may need to set two environment variables for Elasticsearch to work properly:
+```
+xpack.security.enabled: false
+xpack.security.enrollment.enabled: false
+```
+
+And deactivate the `kibana.index` in [kibana.yaml](https://github.com/georchestra/docker/blob/25.0/resources/kibana/kibana.yml#L3).
+
+Example for docker: [here](https://github.com/georchestra/docker/blob/25.0/docker-compose.yml#L365-L366)
+
+## GeoNetwork 4.2.8 to 4.4.8 migration notes
+
+After the upgrade (of Geonetwork and Elasticsearch):
+- Delete index and reindex.
+- JS and CSS cache must be cleared.
+
+You can follow [those `upgrade_geonetwork` steps](https://github.com/georchestra/georchestra/blob/master/docsv1/upgrade_geonetwork.md) to update Geonetwork. 
+
+### Xlinks
+
+> Previously on 4.0.6, it was possible to add contacts to an ISO19139 MD using xlinks, after upgrading to 4.2.2 it wasn't possible anymore.
+
+Since this is merge <https://github.com/georchestra/geonetwork/pull/320>
```

Comment:
> Since this [Pull Request](https://github.com/georchestra/geonetwork/pull/320) is merged, Xlinks contacts are re-enabled by default.

But im not sure about the statement for 4.2.2 .. i had fixed it in https://github.com/georchestra/geonetwork/pull/293 but that never made it to a stable release ?